src/lib/wicked/client.js

```diff
@@ -131,6 +131,21 @@ class WickedClient {
         this._onInterfaceChange.push(fn);
     }
 
+    /**
+     * Whether the wicked service is active or not
+     *
+     * @return {Promise.<boolean>} Promise that resolves to true if service is active or false if not
+     */
+    async isActive() {
+        // FIXME: not using "--value" option here for backward compatibility with systemd < 230
+        // https://lists.freedesktop.org/archives/systemd-devel/2016-May/036583.html
+        const command = 'systemctl show wicked -p ActiveState';
```

Comment:
As discussed in a meeting with @imobachgs and  @teclator , probably is better to use `systemctl --system is-active wicked` instead.